pkg/build/scripts/modify-raw-image.sh.tpl

```diff
@@ -4,28 +4,28 @@ set -euo pipefail
 #  Template Fields
 #  OutputImage   - Full path to the image to modify
 #  CombustionDir - Full path to the combustion directory
-
-#  Guestfish Commands Explanation
-#
-#  sh "btrfs property set / ro false"
-#  - Enables write access to the read only filesystem
-#
-#  copy-in __.CombustionDir__ /
-#  - Copies the combustion directory into the root of the image
-#
-#  sh "btrfs filesystem label / INSTALL"
-#  - As of Oct 25, 2023, combustion only checks volumes of certain names for the
-#    /combustion directory. The SLE Micro raw image sets the root partition name to
-#    "ROOT", which isn't one of the checked volume names. This line changes the
-#    label to "INSTALL" (the same as the ISO installer uses) so it's picked up
-#    when combustion runs.
+#  ConfigureGRUB - Contains the guestfish command lines to run to manipulate GRUB configuration.
+#                  If there is no specific GRUB configuration to do, this will be an empty string.
 #
-#  sh "btrfs property set / ro true"
-#  - Resets the filesystem to read only
+# Guestfish Command Documentation: https://libguestfs.org/guestfish.1.html
 
 guestfish --rw -a {{.OutputImage}} -i <<'EOF'
+  # Enables write access to the read only filesystem
   sh "btrfs property set / ro false"
+
+  # GRUB configuration (if specified)
+  {{.ConfigureGRUB}}
```

Comment:
What do you think about introducing a conditional to make it more clear? It should let us remove / reduce the comments in the Go code.

```suggestion
  {{ if ne .ConfigureGRUB "" }}
      {{ .ConfigureGRUB }}
  {{ end }}
```
Another option is just `{{ if .ConfigureGRUB }}`.